back-end/models/Table.js

```diff
@@ -4,46 +4,87 @@ const Schema = mongoose.Schema;
 
 // Create Schema
 const TableSchema = new Schema({
-  location: {
+
+  ID:{
```

Comment:
we might not actually need an ID - I think we can use the built-in auto-id from mongoose.

https://mongoosejs.com/docs/guide.html#_id

What do you think @brebory ?

Reply:
Yep, using the built-in id makes sense to me.